src/hmr.ts

```diff
@@ -0,0 +1,17 @@
+import * as log4js from "log4js";
+import config from "./config";
+import * as socketEmitter from "chokidar-socket-emitter";
+
+const logger = log4js.getLogger("hmr");
+const server = socketEmitter({
+  port: config.ports.hmr,
+  path: "src",
+  quiet: true,
+  chokidar: {
+    ignored: /___jb.*$/ // IntelliJ Idea creates and removes a temporary file on each save
```

Comment:
The _safe write_ feature? We might suggest to disable it, since it also may cause problems with jspm's dev-bundle mode.